src/ember-resource.js

```diff
@@ -7,6 +7,14 @@
       getPath = Ember.Resource.getPath,
       set = Ember.set;
 
+  // Determine if we need to be back compatible with Ember < 1.12
+  var requiresEmberComputedPropertyFunction = false;
+  try {
+    Ember.computed({get: function() {}, set: function(key, value) {}});
+  } catch(ex) {
+    requiresEmberComputedPropertyFunction = true;
+  }
+
```

Comment:
The reason to do feature detection rather than checking the version is because at Zendesk, we are patching Ember 1.10 to support CP with getter/setter